src/saw-voice.cpp

```diff
@@ -107,7 +107,7 @@ void SawDemoVoice::step()
     for (int i = 0; i < unison; ++i)
     {
         /*
-         * Use a cubic integrated saw and second deriv it at
+         * Use a cubic integrated saw and second derive it at
```

Comment:
It actually should be deriv as in “take the second deriv” her

Reply:
As in short for derivation ? You can tell I'm not a maths guy 😆

Reply:
Yeah that’s how the algo works, it uses the second derivative of the second integral of a sawtooth to suppress aliasing.